drivers/sensors/mpu60x0.c

```diff
@@ -529,7 +543,15 @@ static inline int __mpu_write_reg(FAR struct mpu_dev_s *dev,
 static inline int __mpu_read_imu(FAR struct mpu_dev_s *dev,
                                  FAR struct sensor_data_s *buf)
 {
-  return __mpu_read_reg(dev, ACCEL_XOUT_H, (uint8_t *) buf, sizeof(*buf));
+  if (dev->fifo_enabled)
+    {
+      return __mpu_read_reg(dev, FIFO_R_W, (uint8_t *) buf, sizeof(*buf));
```

Comment:
```suggestion
      return __mpu_read_reg(dev, FIFO_R_W, (uint8_t *)buf, sizeof(*buf));
```

Reply:
I also removed that else bracket, really wasn't necessary.